noinst/specs/compare_specs_csv.R

```diff
@@ -0,0 +1,11 @@
+library(tidyverse)
```

Comment:
Seems outdated, since there is no `_old.csv`? Could be removed?

I think it would be nice if running `specs_from_pdf.R` produces something like `SPECS_DIFF.csv`, with the changes compared to the older `SPECS.csv` that it overwrites.